scripts/make_release_branch.py

```diff
@@ -13,3 +60,10 @@
     git_command("add", GRADLE_PROPERTIES)
     git_command("commit", "-m", ":arrow_up: patch version")
     git_command("push", "origin", "master")
+
+    try:
+        send_slack_message(json.loads(args.slack_ids), patch_version, args.slack_webhook)
+    except Exception as e:
+        print("Problem with Slack message")
+        print(e)
+        print(e.__traceback__)
```

Comment:
It still doesn't look like a proper way to print traceback

I'm not an expert in Python, but chatGPT suggests the follwoing

<img width="793" alt="image" src="https://user-images.githubusercontent.com/2539310/228777455-d6c95bb9-db26-44bb-a795-fa3f109d2784.png">

Looks reasonable enough